src/k8s-engine/lima.ts

```diff
@@ -1763,14 +1763,25 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
   }
 
   async factoryReset(): Promise<void> {
-    const pathsToDelete = [paths.cache, paths.appHome, paths.config, paths.logs];
-
-    if (!pathsToDelete.some( dir => paths.lima.startsWith(dir))) {
+    const promises: Array<Promise<void>> = [];
+    const pathsToDelete = new Set([
+      paths.cache,
+      paths.appHome,
+      paths.altAppHome,
+      paths.config,
+      paths.logs,
+    ]);
```

Comment:
I know it doesn't belong into this PR, but the list is also missing the cache directory for the `rancher-desktop-updater` (see #1684), and I'm wondering where/how that directory name is being constructed.

Reply:
Ugh, right, but at least it's clearly a separate PR :D

(It's based on some Electron-internal logic, I think)